src/capi_frontend/capi.cpp

```diff
@@ -76,7 +76,10 @@ using std::chrono::microseconds;
 extern "C" {
 #endif
 
-OVMS_Status* OVMS_ApiVersion(uint32_t* major, uint32_t* minor) {
+#define DLL_PUBLIC __attribute__((visibility("default")))
+#define DLL_LOCAL __attribute__((visibility("hidden")))
```

Comment:
Why do we need those attributes?
What's more, it seems we do not even use DLL_LOCAL

Reply:
This is to export symbols for external CAPI calls as so library.

Reply:
Why didn't we need it earlier?
Why do you defined DLL_LOCAL if you are not using it anywhere?

Reply:
Its because we set the visibility  for the whole ovms_shared as hidden - python required. I have added the second macro so that we have them both if we need it. It also gives background about what it can be used for.